script/src/verify.rs

```diff
@@ -934,7 +940,7 @@ impl<DL: CellDataProvider + HeaderProvider + ExtensionProvider + Send + Sync + C
         &self,
         group: &ScriptGroup,
         max_cycles: Cycle,
-        snap: &Option<(Snapshot, Cycle)>,
+        snaps: Option<&[(Snapshot, Cycle, ResumePoint)]>,
```

Comment:
suggest to change use slice directly without an Option wrapper, an empty slice means no snapshoots:
```suggestion
        snaps: &[(Snapshot, Cycle, ResumePoint)],
```

Reply:
Done